inc/credential.class.php

```diff
@@ -134,6 +134,9 @@ public function rawSearchOptions()
     public function showForm($ID, array $options = [])
     {
         $this->initForm($ID, $options);
+
+        $this->fields['password'] = (new GLPIKey())->decrypt($this->fields['password']);
```

Comment:
I don't think it's still necessary now that it's no longer on display?

Reply:
indeed